src/roles/vsd-dbbackup/tasks/main.yml

```diff
@@ -53,13 +53,45 @@
     - inventory_hostname in groups['vsds']
     - not vsd_inplace_upgrade
 
+- block:
+
+  - name: find pre upgrade backup script
+    include_role:
+      name: common
+      tasks_from: find-file.yml
+    vars:
+      dir_name: "{{ nuage_unzipped_files_dir }}"
+      pattern: "pre-upgrade-backup.*"
+    delegate_to: localhost
+    
+  - name: VSD pre upgrade backup script location
+    set_fact:
+      image_path: "{{ file_path }}"
+      image_file_name: "{{ file_name }}"
+      
+  - name: Copy and send preupgrade check script to vsd 
+    copy:
+      src: "{{ image_path }}/{{ image_file_name }}"
+      dest: "/opt/vsd/data/"
+    delegate_to: localhost
+    remote_user: "{{ vsd_default_username }}"
+    become: yes
+    
+  - name: Run silent preupgrade check script in the vsd
+    script: "/opt/vsd/data/pre-upgrade-backup.sh -s"
+    remote_user: "{{ vsd_custom_username | default(vsd_default_username) }}"
+    become: yes
```

Comment:
Do we need become_pass here? Can you run a test with hardening and see if it passes. Either of the two 
1. We dont need become at all
2. We need a become_pass